lib/ui/pages/dashboard.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/dashboard_controller.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+
+class Dashboard extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        child: GetBuilder<DashboardController>(
+          builder: (_) {
```

Comment:
I think it would be better to give the variable a name since `_` is usually only used for unused variables or something that could be treated as "native" like `lodash` in Javascript. However, I don't think that's the case here... Or perhaps is this something common when using GetX?

Reply:
fixed in the following PR. I changed it before i had a look at this. Now it'll result in a bunch of merge conflicts

Reply:
okay, let's keep this comment unresolved then and mention in the next PR